File: hidden-api/src/main/java/android/content/pm/PackageInfoHidden.java
package android.content.pm;

import dev.rikka.tools.refine.RefineAs;

@RefineAs(PackageInfo.class)
public class PackageInfoHidden {

  public String overlayTarget;

  public boolean isOverlayPackage() {
    throw new RuntimeException("Stub");
  }
}

File: hidden-api/src/main/java/android/content/pm/PackageInfoHidden.kt
package android.content.pm

import dev.rikka.tools.refine.RefineAs

@RefineAs(PackageInfo::class)
class PackageInfoHidden {

  var overlayTarget: String? = null

  fun isOverlayPackage(): Boolean {
    throw RuntimeException("Stub")
  }
}